src/components/AdminRegisterPage/index.jsx

```diff
@@ -26,14 +28,14 @@ const AdminRegisterPage = ({ authentication, match }) => {
         // user is authenticated but doesn't have the `enterprise_admin` JWT role; force a log out so their
         // JWT roles gets refreshed. on their next login, the JWT roles will be updated.
         const logoutRedirectUrl = getProxyLoginUrl(enterpriseSlug);
```

Comment:
Does it make sense to update this variable name for readability? I really don't understand at first glance how the proxy login url is also the logout url, nor how adding a url to a history forces a log out of any kind.

Reply:
Using the `apiClient` from frontend-auth previously provided a `logout` function that would redirect the user to logout page, and setting the `?next=` query param to whatever URL is passed in as an argument.

Essentially, the behavior we wanted here is largely when the user logs out, they should be brought to the branded logistration page for their Enterprise and this is done by passing `?next=` query param.

Reply:
@raq929  That said, frontend-platform provides a `getLogoutRedirectUrl` that should likely be used here instead:

```
const proxyLoginUrl = getProxyLoginUrl(enterpriseSlug);
const logoutUrl = getLogoutRedirectUrl(proxyLoginUrl);
```

https://edx.github.io/frontend-platform/module-Auth.html#~getLogoutRedirectUrl

Reply:
using `history.push()` is the most common React pattern for updating a url. I agree it's uh, poorly named.